code/modules/mob/living/carbon/human/dummy.dm

```diff
@@ -82,22 +82,27 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 /mob/living/carbon/human/dummy/log_mob_tag(text)
 	return
 
+/proc/get_consistent_feature_entry(list/list)
+	var/consistent_entry = (list - SPRITE_ACCESSORY_NONE)[1]
+	ASSERT(!isnull(consistent_entry))
+	return consistent_entry
```

Comment:
Can you document this proc because It took me a long time to realize what this was actually doing
Also rename `list` to describe what the list actually is

Reply:
```suggestion
/// Takes in an accessory list and returns the first entry from that list, ensuring that we dont return SPRITE_ACCESSORY_NONE in the process.
/proc/get_consistent_feature_entry(list/accessory_feature_list)
	var/consistent_entry = (accessory_feature_list- SPRITE_ACCESSORY_NONE)[1]
	ASSERT(!isnull(consistent_entry))
	return consistent_entry
```